app/models/middleware_server.rb

```diff
@@ -5,4 +5,50 @@ class MiddlewareServer < ActiveRecord::Base
   has_many :middleware_deployments, :foreign_key => "server_id", :dependent => :destroy
   serialize :properties
   acts_as_miq_taggable
+
+  include LiveMetricsMixin
```

Comment:
I noticed all these changes are living on MiddlewareServer...I think it would be better if they all lived on the Hawkular specific subclass.